doc/security/vulnerabilities.rst

```diff
@@ -47,3 +47,28 @@ This has been fixed in releases v1.14.2, v2.2.0, and v2.1.1.
 
 - `PR fix in 2.2.0
   <https://github.com/zephyrproject-rtos/zephyr/pull/23190>`_
+
+CVE-2020-10021
+--------------
+
+Integer Underflow (Wrap or Wraparound), Improper Restriction of
+Operations within the Bounds of a Memory Buffer vulnerability in the
+USB Mass Storage component of zephyrproject-rtos/zephyr allows a
+USB-connected device to read and possibly write system memory,
+possibly resulting in remote code execution.
+
+This has been fixed in releases v1.42.2, v2.2.0, and v2.1.1.
```

Comment:
s/1.42.2/1.14.2/

Reply:
Thanks. I'm redoing this after developing a script to pull this information out of JIRA. Hopefully it will get fixed by that.